backend/src/helpers/statuses.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../models/status')
+
+exports.getStatusByStatusString = async function (statusString) {
+  try {
+    const response = await db.Status.query()
+      .select()
```

Comment:
I think you can use `findOne` instead of `select`. It should give back the item itself as opposed to an array.

Reply:
good advice, done, thank you!